include/zephyr/net/http/service.h

```diff
@@ -99,8 +99,8 @@ struct http_service_desc {
 		.fd = &_name##_fd,                                                                 \
```

Comment:
Please change the commit subject to `net: http: ...`
No need for putting `drivers: ` prefix as the commit is not related to drivers.

Reply:
Done